src/dialects/mariadb/query-generator.js

```diff
@@ -48,6 +51,34 @@ export class MariaDbQueryGenerator extends MySqlQueryGenerator {
     ]);
   }
 
+  removeIndexQuery(tableName, indexNameOrAttributes, options) {
```

Comment:
Can you move this into a `query-generator-typescript.ts` file as well?

Reply:
Done